recipes/atpbar/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "atpbar" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d6fc797cddb96088d001cc4660450b083587a1a1f6eb99f9f02923d2d0987da3
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
```

Is `ipywidgets` (or something similar) an optional runtime dependency? If so, should the version be constrained with:
```suggestion
    - python
  run_constrained:
    - ipywidgets >=X.Y
```
This means `ipywidgets` isn't a required runtime dependency, but if it is installed it has be be version X.Y or greater.

(If the version doesn't matter this can be ignored)

---

Yes if `atpbar` is used on jupyter notebook. In travis-ci, in python 3, `ipywidgets` and `widgetsnbextension` are automatically installed when `jupyter` is installed; however, in python 2.7, they won't be automatically installed with `jupyter`.